packages/context/src/objects/objects.ts

```diff
@@ -323,9 +322,9 @@ export type ActionInput = {
         when?: string;
         eq?: string | number | boolean;
     };
-    property?: {
-        id: string;
-    };
+    property?: InputParameter;
+    viewLayout?: ViewLayoutEntityReference;
```

Comment:
Why is this being added to the legacy `inputProperties`?  When is this ever set?

Reply:
We have a function in Configuration and Widget repos to convert our form to form io components. The function returns a list of ActionInput components. To access `viewLayout` in those components, I need to add to `ActionInput` type.